src/test/kotlin/net/revanced/patcher/PatcherTest.kt

```diff
@@ -44,8 +45,11 @@ internal class PatcherTest {
 
     @Test
     fun testPatcher() {
-        val testData = PatcherTest::class.java.getResourceAsStream("/test1.jar")!!
-        val patcher = Patcher(testData, testSigs)
+        val patcher = Patcher(
+            PatcherTest::class.java.getResourceAsStream("/test1.jar")!!,
+            ByteArrayOutputStream(),
```

Comment:
Tests need to be added for this output. You don't have to make ***this test*** now, it can be added later. (the test for no changes needs to be added, though)